src/main/java/com/stripe/model/Card.java

```diff
@@ -34,6 +34,12 @@
 	String status;
 	String tokenizationMethod;
 
+	// Please note that these field are for internal use only and are not typically returned
+	// as part of standard API requests.
+	String description;
```

Comment:
Way outside of the scope of this PR but as we're following the getters / setter idiom we should declare these as private.